packages/amplication-server/src/core/git/git.resolver.ts

```diff
@@ -68,7 +68,7 @@ export class GitResolver {
   )
   async deleteGitRepository(
     @Args() args: DeleteGitRepositoryArgs
-  ): Promise<Resource> {
+  ): Promise<boolean> {
```

Comment:
why did you decide to change the return type?

Reply:
From now on, which Resource to return?
Git repository have several resources.
So I checked the usage of this endpoint and tested the I don't change existing behavior.